OPAL/br/src/main/scala/org/opalj/br/analyses/cg/InitialEntryPointsKey.scala

```diff
@@ -64,6 +64,11 @@ object InitialEntryPointsKey extends ProjectInformationKey[Iterable[Method], Not
 
         val fqn = entryPointFinder.get
         val epFinder = instantiateEntryPointFinder(fqn)
+
+        for (requiredProjectInformationKey <- epFinder.requirements(project)) {
```

Comment:
This should not be needed, entry point finders that need a key can (and will obviously) get it themselves. However, what is required is that the InitialEntryPointKey's `requirements` method returns the entry point finder's requirements. Have a look at the CallGraphKey, it works pretty much the same